scripts/m2_cromwell_tests/README.md

```diff
@@ -0,0 +1,26 @@
+# M2 Automated Tests for WDL
+
+**This directory is for GATK devs only**
+
+This directory contains scripts for running M2 WDL tests in the automated travis build environment.
+
+# FAQ
+## Does this test Oncotator?
+
+Yes, BUT it tests it without any datasources.
+
+## Does this test FilterByOrientationBias?
+
+Yes
+
+## Does this test the GATK4 jar override parameter (gatk4_jar_override)?
+
+No
+
+## Does this test filering?
```

Comment:
typo: `filering`.  Also, specify that this is filtering common germline variants, since strand bias, normal artifact, etc. are also filters.